apps/arkmarket/src/app/collection/[collectionAddress]/components/desktop-collection-activity.tsx

```diff
@@ -206,7 +206,7 @@ export default function DesktopCollectionActivity({
               </TableCell>
               <TableCell className="pr-5">
                 <Button asChild size="icon" variant="outline">
-                  <ExternalLink href="/">
+                  <ExternalLink href={`https://starkscan.co/nft/${activity.address}/${activity.token_id}`}>
```

Comment:
Hey @stephanniegb, thanks for your contribution

This link should also point to the transaction itself, as you did in `desktop-portfolio-activity`